### ISSUE TITLE
Guidelines for using the brand in meetups

### ISSUE THREAD

**Issue by tobie** [w3c/testtwf-website#131]

**Comment by rhauck** [comment]
Suggestion: When you do this, it'd be nice to provide some downloadable assets for organizers who want to make tshirts, stickers, posters, etc.

**Comment by tobie** [comment]
Agreed. Do we have such assets anywhere? Care to help me set that section up?

**Comment by rhauck** [comment]
Yes and yes. 

I'm collecting them now and making vectors where needed.  I'll stage it all on Dropbox so you can see what we've got and decide what should be included. For example, I've got files for stickers, posters and tshirts that can be sent to vendors as-is without having to recreate them.  They're a little disorganized right now, so I'll have them ready in a day or two.
